.circleci/config.yml

```diff
@@ -1,83 +1,197 @@
 jobs:
   build_centos7:
     docker:
-     - image: centos:7
-
-    working_directory: ~/warewulf3
-
+      -
+        image: "centos:7"
     steps:
       - checkout
-      - run:
-          name: Install Build Dependencies
+      -
```

Comment:
This is just a nit, but there's a handful of instances where there's a list `- ` with a newline. Can the extra newline be removed. In this instance for example, `run:` should be `- run:`.

Reply:
I copy/pasted the yamllint output. I prefer it your way too. The linter rule is that a key/value in a sequence is on the same line, with the exception that a block mapping in a sequence gets a newline after the hyphen.
Let me know if we need to fix it, although I do like the ability for auto-formatters do the work for me.

Reply:
If yamllint likes it I guess its good enough for me too. Thanks.